### ISSUE TITLE
Undo removes multiple execution listeners at a time

### ISSUE THREAD

**Issue by pinussilvestrus** [camunda/camunda-modeler#3016]
### Describe the bug

When I add multiple execution listeners to a C7 BPMN diagram and press CMD+Z to undo, it removes multiple (not all) listeners and not only the last one added.

![Kapture 2022-07-08 at 11 17 38](https://user-images.githubusercontent.com/9433996/177961148-e854b7bd-719b-4595-bead-75737e400f7c.gif)

### Steps to reproduce

1. Select a service task
2. Add multiple execution listeners
3. Undo changes (e.g. via CTRL/CMD+Z)
4. See multiple listeners were removed, not only the last one added

### Expected behavior

Only the last added listener should be removed in this scenario. 

Note that it works as expected for other list groups (e.g. Inputs and Outputs).

### Environment

- OS: MacOS 11.6.5
- Camunda Modeler Version: 5.1.0-rc.1, 5.0.0
- Execution Platform: Camunda Platform 7.15
- Installed plug-ins: Cloud Connect Modeler Plugin

**Comment by pinussilvestrus** [comment]
Note that `redo` also brings back all removed listeners. It seems like the command stack is combining multiple add operations in this case.

**Comment by pinussilvestrus** [comment]
Moving this to the backlog and adding the spring cleaning label as it is not a blocker.

**Comment by pinussilvestrus** [comment]
> I can reproduce this in the Modeler but not in bpmn-js-properties-panel or camunda-bpmn-js 🤔

That could be an issue with the modeling behaviors then: https://github.com/camunda/camunda-bpmn-js-behaviors

**Comment by philippfromme** [comment]
I'll try my luck. 🤡

**Comment by philippfromme** [comment]
# tl;dr
Properties panel keyboard events aren't stopped from being propagated and therefore are handled twice. Only thanks to a quirk does this not result in undo being triggered twice in most cases.

# Root Cause Analysis

* there are two places in the Camunda Modeler where we trigger undo and redo
  * [`KeyboardBindings`](https://github.com/camunda/camunda-modeler/blob/v5.2.0/client/src/app/KeyboardBindings.js#L130) - triggers undo and redo if edit menu contains entries and they're enabled
  * [`PropertiesPanelKeyboardBindings`](https://github.com/camunda/camunda-modeler/blob/v5.2.0/client/src/app/tabs/bpmn/modeler/features/properties-panel-keyboard-bindings/PropertiesPanelKeyoardBindings.js#L56) - triggers command stack undo and redo instead of native when properties panel container is focused

The _expected_ behavior when working with the properties panel is:

* the properties panel is focused e.g. an input is focused
* when <kbd>Ctrl</kbd> + <kbd>Z</kbd> is pressed undo is triggered by `PropertiesPanelKeyboardBindings`
* no other keyboard listeners are triggered

The _actual_ behavior when working with the properties panel is:

* the properties panel is focused e.g. an input is focused
* when <kbd>CMD</kbd> + <kbd>Z</kbd> is pressed undo is triggered by `PropertiesPanelKeyboardBindings`
* the event is propagated and undo is triggered again by `KeyboardBindings`

The reason why this behavior can only be observed when adding execution listeners is that when adding an execution listener no input of the execution listener will be focused.

![electron_6sElfuYKAc](https://user-images.githubusercontent.com/7633572/185077609-42709265-3c4b-420b-a002-1b4047ad1b18.gif)

Compare that to adding inputs.

![electron_cVMW7LurSZ](https://user-images.githubusercontent.com/7633572/185077912-434d611a-eaba-40d9-9970-a3239bf9ab14.gif)

When focusing a properties panel input the following happens:

* edit menu is updated
* since an input is focused native undo and redo entries are used (cf. https://github.com/camunda/camunda-modeler/blob/v5.2.0/client/src/app/tabs/bpmn/BpmnEditor.js#L397)
* when pressing <kbd>Ctrl</kbd> + <kbd>Z</kbd> `KeyboardBindings` will _not_ trigger command stack undo since native undo and redo entries are used

Since adding an execution listener doesn't focus an input the edit menu isn't updated with the native undo and redo entries. So it is only thanks to the edit menu update that in most cases undo isn't triggered twice.

What's missing in `PropertiesPanelKeyboardBindings` is to stop the event from being propagated. This was probably intended but forgotten to implement. `preventDefault` is called but `stopPropagation` isn't (cf. https://github.com/camunda/camunda-modeler/blob/develop/client/src/app/tabs/bpmn/modeler/features/properties-panel-keyboard-bindings/PropertiesPanelKeyoardBindings.js#L59).

**Comment by philippfromme** [comment]
Fixed by https://github.com/camunda/camunda-modeler/pull/3095. Somehow `Closes #xxxx` doesn't seem to work anymore. 🤔
